### PR TITLE
docs: update README, dataflow and deployment

### PR DESCRIPTION
**Pull request by tm-jarl-briones**

### CODE REVIEW

**Review by tm-echo-arcelao — APPROVED** [review]

File: docs/deployment.md [hunk 1]
```diff
@@ -1,18 +1,124 @@
...
-## Dev Deployment
+## Deployment Steps
 
-### Pre-requisites
-``` @TODO: Fill with pre-reqs such as access, CI/CD setup, variables used for deployment etc ```
+1. **AWS Infrastructure Setup**
 
-### How-to-Guide
-``` @TODO: Fill with steps to deploy. Feel free to subdivide to sections or multiple MD files through mkdocs.yml ```
+Used Terraform with the de-workflow-template to create the AWS infrastructure.
```
Comment (tm-echo-arcelao) [inline docs/deployment.md:7]: you may copy over the directions from the [`dwt-terraform-template`](https://github.com/thinkingmachines/dwt-terraform-template/tree/main/aws/terraform) and add them here

File: docs/deployment.md [hunk 2]
```diff
@@ -1,18 +1,124 @@
...
+    - Default Action: Forward to dwh-target-group
+
+  Wait until the Load Balancer is Active. It should have a DNS that you can use to access Dagster using your browser.
+
+## Post-Deployment
+
+- Verify that the Dagster UI is accessible via the Load Balancer's DNS.
+- Test the deployed application to ensure all components are functioning correctly.
+
+DNS name: http://dwh-alb-jjk-963539840.ap-southeast-1.elb.amazonaws.com
```
Comment (tm-echo-arcelao) [inline docs/deployment.md:118]: assume this is inaccessible as we will take down the resources after the lab.

File: docs/deployment.md (hunk 1 above)
Comment (tm-echo-arcelao) [inline docs/deployment.md:7]: you may copy over the directions from the [dwt-terraform-template](https://github.com/thinkingmachines/dwt-terraform-template/tree/main/aws/terraform) and add them here

File: docs/deployment.md
Comment (tm-echo-arcelao) [inline docs/deployment.md:1]: add directions to configure `buildspec.yml` and `scripts/start-process.sh` as well

File: docs/README.md [hunk 3]
```diff
@@ -1,15 +1,222 @@
-# GitHub Starter
+# Data Platform on Cloud Lab Exercise
 
-## Project Brief
+This lab exercise outlines the steps for deploying a containerized data platform project from the Data Platform Exercise onto the AWS TM-TALENT-ENGINE sandbox. This deployment will leverage a combination of AWS services, such as EC2, S3, RDS, and Load Balancing, as discussed in the Data Platform on Cloud session.
```
Comment (tm-kyle-naranjo) [inline docs/README.md:3]: add description of what the dagster app does/refresher on the data platform lab project brief

File: docs/deployment.md [hunk 4]
```diff
@@ -1,18 +1,124 @@
...
-``` @TODO: Fill with steps to deploy. Feel free to subdivide to sections or multiple MD files through mkdocs.yml ```
+  - Project Name: dwh-codebuild-jjk
+  - Source: GitHub
+  - Credential: Custom Source Credential to override account's default settings
+  - Connection:
+    - If not yet connected:
+      - Create a new GitHub Connection
+    - Connection Name: dwh-github-connection-jjk
+    - Repository: Select the repo containing Dagster files
+    - Webhook: Rebuild on code push (Event Type: Push) (This was not implemented in this project as we experienced a Github API limit error.)
```
Comment (tm-kyle-naranjo) [inline docs/deployment.md:24]: I think we were able to resolve the API limit, but just went with the manual approach

File: docs/deployment.md [hunk 5]
```diff
@@ -1,18 +1,124 @@
...
+    - dwh-vpc-staging-jjk-public-us-west-2a
+    - dwh-vpc-staging-jjk-private-us-west-2b
+  - Listeners:
+    - Protocol: HTTP
+    - Port: 80
+    - Default Action: Forward to dwh-target-group
+
+  Wait until the Load Balancer is Active. It should have a DNS that you can use to access Dagster using your browser.
+
+## Post-Deployment
```
Comment (tm-kyle-naranjo) [inline docs/deployment.md:113]: Please add teardown

File: docs/deployment.md (hunk 1 above)
Comment (tm-jarl-briones) [reply docs/deployment.md:7]: i reference the template nlg for this. so that it will not look very long and we can focus lang on our deployment

Comment (tm-echo-arcelao) [reply docs/deployment.md:7]: copy. good with me!

**Review by tm-kyle-naranjo — APPROVED** [review]

**Pull request merged**